### DISCUSSION TITLE
Character first structure: browse Unicode character first, then see font variations

### DISCUSSION THREAD

**Discussion by TheJoeFin** [character-map-uwp/Character-Map-UWP#290] [Ideas]
Today UWP Character Map is a font first structure. I am wondering if there is a way to have an option, maybe in the font filter drop down which inverts the structure to be character based. What does this mean:

1. Filter by Unicode groups
2. View all Unicode characters
3. See which characters are present in different fonts
    - This would be mostly useful for the special Unicode characters
    - This could be a challenging to do the link

Example Scenario:
Looking for arrow characters to use in an app, I would like to be able to browse all of the Unicode arrows and see which one fits my design best.

**Comment by JohnnyWestlake** [comment]
I might consider something at some point, but in your example scenario, you can open Font Compare View (App menu -> Compare Fonts, or Ctrl+K) and put the arrows in the preview box at the top and you'll be able to see what they look like in every font.

**Reply by TheJoeFin** [reply]
Yes, the Compare Fonts definitely works well! I tested out my scenario and found many fonts do not have the arrow characters, so there were a lot of empty boxes. Also, there are several very similar arrows in Unicode which makes it kind of a pain to scan through and find the right one. I usually use (Win + .) to find something close then go through different fonts to refine it. Not a great workflow today.

**Comment by JohnnyWestlake** [comment]
Update: there will be very basic support for character filtering in the next update in the main font window by typing in `char: ` followed by your query in the font list search box. 

i.e, to find all fonts containing `←` & `↗` you would type in `char: ←↗`.

![image](https://github.com/character-map-uwp/Character-Map-UWP/assets/2319473/c1b6740e-9ade-4823-8b7e-efe01dda4487)

I will try to work out a good UI for exposing this in the update after, including support for Compare fonts view.
